builder-support/debian/authoritative/debian-buster/pdns-backend-mysql.NEWS

```diff
@@ -0,0 +1,9 @@
+pdns (4.1.3-4) unstable; urgency=medium
```

Comment:
I'm not sure we should ship this in our packages, idem for the ones in gpgsql and gsqlite3

Reply:
your call :)

Reply:
let's drop them then :)

Reply:
should I also drop pdns-server.NEWS?

Reply:
yes please

Reply:
done!